BadgeFramework/badge_protocol.py

```diff
@@ -10,37 +10,35 @@
 Request_identify_request_tag = 27
 Request_restart_request_tag = 29
 Request_free_sdc_space_request_tag = 30
-Request_sdc_errase_all_request_tag = 31
-Request_get_imu_data_request_tag = 33
 
 Response_status_response_tag = 1
 Response_start_microphone_response_tag = 2
 Response_start_scan_response_tag = 3
 Response_start_imu_response_tag = 4
 Response_free_sdc_space_response_tag = 5
-Response_sdc_errase_all_response_tag = 32
-Response_get_imu_data_response_tag = 34
+
```

Comment:
I don't know what happened with this file but I think all the changes should be removed